downstream/modules/platform/con-aap-upgrade-planning.adoc

```diff
@@ -0,0 +1,30 @@
+
+
+[id="aap-upgrade-planning_{context}"]
+
+= {PlatformNameShort} upgrade planning
+
+[role="_abstract"]
+Before you begin the upgrade process, review the following considerations to plan and prepare your {PlatformNameShort} deployment:
+
+[discrete]
+== {ControllerNameStart}
+
+* Even if you have a valid license from a previous version, you must provide your credentials or a subscriptions manifest upon upgrading to the latest version of {ControllerName}.
+* If you need to upgrade Red Hat Enterprise Linux and {ControllerName}, you must first backup and restore of your {ControllerName} data.
```

Comment:
Unless I'm missing info, I think there is a typo in "restore of". It should be just "restore". Or should it be "you must first do a backup and restore of..."?